Updated nsfw-js, tensorflowjs and puppeteer to the latest version, fixed tests hanging.

gboere opened the pull request:
#### Changes
- Updated the nsfw-js, puppeteer and tensorflow dependencies to their latest versions.
- I'm marking this as a breaking change because it introduces three major version updates to the dependencies, but it builds with almost no change to the original code.
- Tests were hanging due to imgur redirect, so I changed the urls to imghippo.
- Building now requires a temporary fix due to dependencies using outdated  an outdated version of npm's ssl library.

Fixes #261 

#### Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

gboere committed:
bumped nsfw-js to @latest

gboere committed:
changed test urls, deleted unrelated files from last commit

pottekkat commented:
@gboere Thank you for your PR. This updates would be a good first step towards migrating to manifest v3. Could you also update the package-lock.json file as the CI tests are failing.

I apologize for the slow review. I'm the sole maintainer of this project now and I don't get enough time to work on this.

gboere committed:
Updated webpack, antd, related libs.

gboere committed:
More antd stuff.

gboere committed:
Finally, updated antd.

gboere committed:
Jest upgrade:
Fixed some tests, but others aren't working.
Needs rewriting.

Also fixed Model version (v2).
